server.go

```diff
@@ -124,6 +124,7 @@ func NewServer(options ...func(*Server)) *Server {
 		WithRouteOptions(
 			OptionAddResponse(http.StatusBadRequest, "Bad Request _(validation or deserialization error)_", Response{Type: HTTPError{}}),
 			OptionAddResponse(http.StatusInternalServerError, "Internal Server Error _(panics)_", Response{Type: HTTPError{}}),
+			OptionHeader("Accept", "Accept header parameter"),
 		),
 	}
```

Comment:
Here you're adding the parameter to all routes.

Fuego can also register standard `func(w http.ResponseWriter, r *http.Request)` routes that don't need this option.

I suggest that you move this to the `registerFuegoController()` function in `net_http_mux.go` file